v3/setup/etcdctl-auth.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+source /etc/environment
+
+#Disable authentication
+
+#curl  -L http://127.0.0.1:2379/v2/auth/enable -XDELETE
+
+etcdctl set /etcdctl/config/root-user root
+etcdctl set /etcdctl/config/read-user read-user
+etcdctl set /etcdctl/config/write-user write-user
+
+ROOT_USERNAME=$(etcdctl get /etcdctl/config/root-user)
+ROOT_PASSWORD=$(etcdctl get /etcdctl/config/root-password)
+READ_USERNAME=$(etcdctl get /etcdctl/config/read-user)
+READ_PASSWORD=$(etcdctl get /etcdctl/config/read-password)
+WRITE_USERNAME=$(etcdctl get /etcdctl/config/write-user)
+WRITE_PASSWORD=$(etcdctl get /etcdctl/config/write-password)
+
+CRED_DIR="/opt/etcdctl"
+if [[ ! -d $CRED_DIR ]]; then
+    sudo mkdir $CRED_DIR -p
+fi
+
+#add a root user
+
+sudo echo '{"user":"'$(eval echo $ROOT_USERNAME)'", "password":"'$(eval echo $ROOT_PASSWORD)'"}' > $CRED_DIR/root-user.json
```

Comment:
I don't think you need the `eval`. Actually `"'${ROOT_PASSWORD}'"` might just work.